packages/widget/src/pages/SwapExecutionPage/useSwapExecutionState.ts

```diff
@@ -38,7 +38,7 @@ export function useSwapExecutionState({
       return SwapExecutionState.confirmed;
     }
 
-    if (overallStatus === "pending") {
+    if (overallStatus === "pending" || overallStatus === "failed") {
```

Comment:
this is so the edge case of us waiting the extra 5 seconds we continue to show "pending" in the `MainButton`, currently we show `Validating` if there's an error